package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "kaltura-player-js",
   "version": "0.55.0",
-  "main": "dist/kaltura-player.js",
+  "main": "dist/kaltura-ovp-player.js",
```

Comment:
why did you change the main?

Reply:
Because there's no dist file called kaltura-player.js